tutorials/ner_double/configs/drugs_second.cfg

```diff
@@ -0,0 +1,58 @@
+[paths]
+train = null
+dev = null
+
+[system]
+gpu_allocator = null
+
+[nlp]
+lang = "en"
+# Note this is the only line that's different from the other config!
+pipeline = ["tok2vec","ner","ner_drug"]
+batch_size = 1000
+
+[components]
+
+[components.tok2vec]
+source = "en_core_web_md"
```

Comment:
You don't need this, the `ner` has an internal `tok2vec`.

Reply:
Ah, of course!